README.md

```diff
@@ -139,6 +139,10 @@ the instance id with `#{id}`.
 ### With Capistrano
 
 ```ruby
+#./ Capfile
```

Comment:
```suggestion
# ./Capfile
```

also, I don't think it matters but in our projects I've always seen it put in `deploy.rb` - I'm mentioning it more as I'm curious if there is ever a situation where `Capfile` would be loaded but not `deploy.rb`?